### PR TITLE
[1.4 - DO NOT MERGE] Add GKE section for Stratos

### LINKED ISSUE #407: Stratos: update deployment and connect to kube for all platforms
A GKE section for Stratos 2.4/CAP 1.4
Add deploying Stratos to GKE deployment
Update "Connect Kubernetes to Stratos" for CaaSP, AKS, EKS, GKE

### PR DESCRIPTION
**Pull request by btat**
Resolves #407

### CODE REVIEW

File: xml/cap_depl_stratos.xml [hunk 1]
```diff
@@ -230,6 +230,36 @@ gp2scoped       kubernetes.io/aws-ebs   1d
...
+<screen>helm install suse/console \
+    --name susecf-console \
+    --namespace stratos \
+    --set services.loadbalanced=true \
+    --set console.service.http.nodePort=8080
+</screen>
+  <para>
+    Monitor progress:
+  </para>
+<screen>$ watch -c 'kubectl get pods --namespace stratos'</screen>
```
Comment (cjschroder) [inline xml/cap_depl_stratos.xml]: use entity for command prompt

File: xml/cap_depl_stratos.xml [hunk 2]
```diff
@@ -230,6 +230,36 @@ gp2scoped       kubernetes.io/aws-ebs   1d
...
+
+  <para>
+   Before deploying Stratos, ensure <literal>uaa</literal> and <literal>scf</literal>
+   have been successfully deployed on &gke; (see <xref linkend="cha.cap.depl-gke"/>).
+  </para>
+
+  <para>
+   Use &helm; to install Stratos:
+  </para>
+<screen>helm install suse/console \
```
Comment (cjschroder) [inline xml/cap_depl_stratos.xml]: use entity for command prompt

Comment (btat) [reply xml/cap_depl_stratos.xml]: Thanks! Entity added now for all missing instances.

**Comment by cjschroder** [comment]
Not sure why Travis is having a meltdown-- you should email @sknorr

**Comment by btat** [comment]
> Not sure why Travis is having a meltdown-- you should email @sknorr

I based this branch off of develop so it doesn't have the `gke` entity (created in the gke branch) yet. Should be ok afterwards.

**Comment by sknorr** [comment]
Sorry, saw this just now -- probably too little too late but looking at the Travis log, apparently the \&gke; entity was not defined in a .ent file.

**Comment by btat** [comment]
No worries, it's defined in another PR, but I kept forgetting to rebase this one.





Billy Tat
Software Engineerbtat@suse.com

>>> Stefan Knorr <notifications@github.com> 05/06/19 2:52 AM >>>
Sorry, saw this just now -- probably too little too late but looking at the Travis log, apparently the &gke; entity was not defined in a .ent file.

**Pull request merged**